.github/workflows/cron-docker.yaml

```diff
@@ -0,0 +1,26 @@
+name: Create a commit to update SHA tag and dispatch to launch Docker images build
+
+on:
+  schedule:
+    # weekly
+    - cron: "0 0 * * 0"
+
+jobs:
+  build-images:
+    name: Create a commit to update SHA tag and dispatch to launch Docker images build
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Clone Main Repo
+        uses: actions/checkout@v2
+      - name: Create an empty commit
+        uses: stefanzweifel/git-auto-commit-action@5dd17c3b53a58c1cb5eaab903826abe94765ccd6 # dependabot updates to latest release
+        with:
+          commit_message: Empty commit to update sha tag
+          commit_options: "--allow-empty"
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@ce5485de42c9b2622d2ed064be479e8ed65e76f4 # dependabot updates to latest release
+        with:
+          token: ${{secrets.REPO_ACCESS_TOKEN}}
```

Comment:
@parente could you please give me the necessary rights for managing secrets?

Reply:
@parente may I ask you to respons here?

Reply:
@mathbunnyru I never received a GitHub notification for your @ here. Odd. I've updated your and @consideRatio's permissions on this repo to admin from maintainer.

Reply:
Thanks, @parente!
Maybe you have accidentally disabled notifications for the project or for this particular PR?

Anyway, thanks for the admin permissions ❤️ Now, I will be able to make changes like this easier👍